packages/components/stories/DatePicker.stories.tsx

```diff
@@ -139,3 +140,45 @@ export const InForms: Story = {
 		await expect(await within(body).findByRole('application'));
 	},
 };
+
+export const InSubmenu: Story = {
```

Comment:
[storybook](https://626696a2018c1f004a1cde86-etmmhzcksj.chromatic.com/?path=/story/components-date-and-time-datepicker--in-submenu)